app/src/androidTest/java/org/mozilla/fenix/helpers/HomeActivityTestRule.kt

```diff
@@ -62,6 +70,9 @@ class HomeActivityIntentTestRule(
         super.beforeActivityLaunched()
         setLongTapTimeout(3000)
         if (skipOnboarding) { skipOnboardingBeforeLaunch() }
+        if (skipShowJumpBackIn) {
+            activity.settings().shouldShowJumpBackInCFR = false
```

Comment:
question: shouldn't this have the same `activity == null` issue that `HomeActivityTestRule` has? Have we tested this rule to make sure it works properly?